### PR TITLE
Fix obvious bug in particle boosting (recursive statement)

### PR DESCRIPTION
**Pull request by covarell**
The title says it all.
URGENTLY NEEDED for signal 10.2.X MC production

### CODE REVIEW

**Comment by alberto-sanchez** [comment]
please test

File: GeneratorInterface/GenFilters/src/MCParticlePairFilter.cc [hunk 1]
```diff
@@ -230,7 +230,7 @@ bool MCParticlePairFilter::filter(edm::StreamID, edm::Event& iEvent, const edm::
 	     charge1 = charge((*p)->pdg_id());
 	     //totmomentum = momentum1 + typeApassed[i]->momentum();
```
Comment (vargasa) [inline GeneratorInterface/GenFilters/src/MCParticlePairFilter.cc]: Remove commented code if not needed. Fix identation

**Review by vargasa — CHANGES_REQUESTED** [review]

File: GeneratorInterface/GenFilters/src/MCParticlePairFilter.cc (hunk 1 above)
Comment (covarell) [reply GeneratorInterface/GenFilters/src/MCParticlePairFilter.cc]: Hello @vargasa I can but that was not my fault :-)

**Comment by vargasa** [comment]
Hi @covarell. You are requesting to merge this into `master`. Are you writing a backport on `10_2_X`?

**Comment by covarell** [comment]
Indeed. When we converge, will prepare it

**Comment by davidlange6** [comment]
please test

**Comment by vargasa** [comment]
> will prepare it

Sorry, I misstated my question. I did not intend to suggest a backport. 
By your PR description I thought the PR was intended to be against 
10_2_x instead of master. Please ignore my comment.

**Comment by efeyazgan** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**